src/puppet-redirect.html

```diff
@@ -1,6 +1,6 @@
-<link rel="import" href="../puppet-redirect.html">
+<link rel="import" href="../palindrom-redirect.html">
```

Comment:
I would remove this file completely, let's not keep backward compatibility forever.